features/steps/method_summary.py

```diff
@@ -26,6 +26,9 @@ def check_title(context, title):
 def check_ons_meta_data(context, metaDataField, metaDataValue):
     ons_meta_data_fields = extract_ons_meta_data(context)[0]
     ons_meta_data_values = extract_ons_meta_data(context)[1]
+
+    ons_meta_data_values = [val.replace('Release version\n','').replace('\n(opens in a new tab)','') for val in ons_meta_data_values]
```

Comment:
I don't see any new changes within the method summary feature file. So why are we changing the step definitions here?

Reply:
I modified method.html to include two visually hidden spans within the link containing the text 'Release version' and '(opens in a new tab)'. This means that when selenium 'gets' the text content of the link for testing it will receive '<span>Release version</span> v1.0.0 <span>(opens in a new tab)</span>'. The span tags get automatically removed which causes some formatting issues and introduces some whitespace characters, but I can't put any of these in the feature file as they get automatically escaped... What I'm doing here is removing the visually hidden text and any whitespace characters I may have introduced, so that I can leave the feature file unmodified. I'll keep working on this I think there might be a better way